pkg/validation/options.go

```diff
@@ -13,6 +13,8 @@ import (
 	"sort"
 	"strings"
 
+	"github.com/oauth2-proxy/oauth2-proxy/pkg/util"
+
```

Comment:
I don't think we need this new line here

Reply:
Done.